### PR TITLE
Fix alert banner overlap with "Switch views" button

### LINKED ISSUE #12662: UI Bug — "Switch Views" Misaligned when Alert is Present
## Description
The "Switch views" dropdown on Queue page is positioned absolutely within its container, placing it in the upper right. While this looks OK in the default state of that page, things get weird when an alert (success, etc) is present (see screeenshot)

## Acceptance criteria
- [ ] Position of "Switch views" dropdown isn't wonky when alert is present (i.e. retains its default state position when banners appear)

### Screenshot of bug
Screen Shot 2019-11-07 at 10 10 48

### PR DESCRIPTION
**Pull request by yoomlam**
Resolves #12662

### Description
Adjust "Switch views" dropdown button position to prevent overlapping alert banner.

### Acceptance Criteria
- [ ] Code compiles correctly
- [ ] All other pages that have an alert/success banner and a "Switch views" button render as desired.

### Testing Plan
Run through the following steps on https://www.caseflowdemo.com/ and compare the rendering against https://yoom.caseflowdemo.com/
1. Log on as an admin for some organization (i.e., a user that sees the "Switch views" button), e.g., `BVAAABSHIRE`
2. Go to their queue
2. Cause an alert/success banner to occur, e.g., completing an assigned task
3. Compare the page render
4. Repeat with admins from different organizations

### User Facing Changes

 BEFORE|AFTER
 ---|---
Screen Shot 2019-11-07 at 10 10 48 | image

### CODE REVIEW

File: client/app/queue/components/QueueSelectorDropdown.jsx [hunk 1]
```diff
@@ -5,11 +5,8 @@ import { css } from 'glamor';
 import DropdownButton from '../../components/DropdownButton';
 import COPY from '../../../COPY';
 
-// NOTE: parent container needs to be given 'position: relative'
 const style = css({
-  position: 'absolute',
-  top: '48px',
-  right: '40px'
```
Comment (yoomlam) [inline client/app/queue/components/QueueSelectorDropdown.jsx:12]: The original implementation uses `absolute`, which disregards placement of other elements that may overlap with this button. See https://www.w3schools.com/css/css_align.asp.

**Comment by geronimoramos** [comment]
Can we align the dropdown menu to the Your Cases heading? Pictured here in the default state

image

**Comment by meckila-navapbc** [comment]
It looks like there's a `margin-right` coming from the dropdown component that could be set to 0
Screen Shot 2021-01-27 at 12 11 52 PM

File: client/app/queue/components/QueueSelectorDropdown.jsx [hunk 2]
```diff
@@ -5,11 +5,8 @@ import { css } from 'glamor';
...
+  float: 'right'
```
Comment (meckila-navapbc) [inline client/app/queue/components/QueueSelectorDropdown.jsx]: To also align the dorpdown with the tabs, removing margin right should work
```
const style = css({
  float: 'right',
  margin-right: 0
```

**Comment by yoomlam** [comment]
@geronimoramos I nudged it up 40px. The button itself is 37px tall. I redeployed the changes to https://yoom.caseflowdemo.com.

**Comment by yoomlam** [comment]
Redeployed and ready for Designer review and testing.

**Comment by jcq** [comment]
I definitely defer to designers on this one, but my sense is that there should be even margin above and below the alert. By default `.usa-alert` has `margin-top: 1.5em` (guh re "em" as the unit); at very least we should probably try matching that as bottom margin in this context, as the `h1` below it has no top margin.

**Comment by yoomlam** [comment]
@jcq Sounds good. I'll wait for other designer input before updating.

**Comment by mkhandekar** [comment]
I agree with @jcq's feedback! 💯

**Comment by yoomlam** [comment]
I redeployed the changes to https://yoom.caseflowdemo.com. It is ready for thorough designer testing (i.e., ensure that the button is not occluded and can be clicked by users on all relevant web pages).

**Review by geronimoramos — APPROVED** [review]
Looks good!

**Pull request merged**